docs/organizations/changing-your-plan-and-billing.md

```diff
@@ -28,8 +28,8 @@ Choose one of the following options in your organization **Settings**, page **Pl
 
 -   **Allow new people to join immediately:** people with access to the organization on the Git provider can join the organization on the Codacy app immediately, as long as there are seats available.
 
--   **Review join requests from new people:** when people with access to the organization on the Git provider join the organization on Codacy app, an organization owner must manually approve their requests to join on the page **People**.
+-   **Review join requests from new people:** when people with access to the organization on the Git provider join the organization on Codacy app, an organization admin must manually approve their requests to join on the page **People**.
 
-    Your teammates that have already been invited to join or were added to the organization are automatically approved, and you can also skip the approval process for organization owners.
+    Your teammates that have already been invited to join or were added to the organization are automatically approved, and you can also skip the approval process for organization admins.
```

Comment:
This one is a bit tricky because the **admin** here relates to the Git provider, right?
Maybe this will fit the three providers. WDYT? (**admins** fits GitLab and Bitbucket, **organization owners** fits GitHub)

```suggestion
    Your teammates that have already been invited to join or were added to the organization are automatically approved, and you can also skip the approval process for other admins or organization owners.
```

Reply:
Now that I'm re-reading [this entire section](https://docs.codacy.com/organizations/changing-your-plan-and-billing/#allowing-new-people-to-join-your-organization), it doesn't mention that it's GitHub-specific, but the code I linked to above suggests otherwise. Thoughts?

Reply:
AFAIK, this section applies to all Git providers. 🤔 
We should confirm it with the engineering, on the code side.

Reply:
Maybe @inesgomas can help us clarify here? 🙏

Reply:
I confirm that the feature to manage request is not for GitHub only. It's all providers. 

I believe the original suggestion (as in the commit) its the correct one. The moment the user creates a codacy account it already has it's permissions mapped, even though he still has not joined the organization, we did the map in the background and we know if it is only an member or a organization admin (not true for BB, I think, but we do that the moment they click the button, for example, so these stuff continue to happen in the background somehow, not detected by our users). 
This feature is to define if you want those **organization admins** to auto-join or not. Looks good

Reply:
Thanks @inesgomas! Resolving this one.